plugins/monolith/ortac-monolith.opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Monolith plugin got Ortac"
+description: "Monolith plugin got Ortac"
+maintainer: ["Nicolas Osborne <nicolas.osborne@tarides.cmd>"]
+authors: ["Nicolas Osborne <nicolas.osborne@tarides.cmd>"]
```

Comment:
It is `.com`, no (in lines 4 and 5)?

Reply:
Indeed. It was fixed in #69 but missed in some rebasing of this PR it seems ;-)

Reply:
Fixed :-)